addons/sale_timesheet/models/project_update.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+from odoo.tools import format_amount
+from odoo.tools.misc import formatLang
+
+class ProjectUpdate(models.Model):
+    _inherit = 'project.update'
+
+    @api.model
+    def _get_template_values(self, project_id):
+        return {
+            **super(ProjectUpdate, self)._get_template_values(project_id),
+            'profitability': self._get_profitability_values(project_id=project_id),
+        }
+
+    @api.model
+    def _get_profitability_values(self, project_id):
+        today = fields.Date.context_today(self)
+        a_month_ago = fields.Date.context_today(self) + relativedelta(months=-1)
+        two_month_ago = fields.Date.context_today(self) + relativedelta(months=-2)
+        profitability = self.env['project.profitability.report'].read_group(
+            [('project_id', '=', project_id)],
+            ['project_id',
+             'amount_untaxed_to_invoice',
+             'amount_untaxed_invoiced',
+             'expense_amount_untaxed_to_invoice',
+             'expense_amount_untaxed_invoiced',
+             'other_revenues',
+             'expense_cost',
+             'timesheet_cost',
+             'margin'],
```

Comment:
Same remark in addons/sale_timesheet/models/project.py#_get_profitabilities

Reply:
The methods were merged :)